package.json

```diff
@@ -89,6 +91,7 @@
     "react-native-vector-icons": "^10.0.0",
     "react-native-zip-archive": "^6.1.0",
     "react-test-renderer": "^18.2.0",
+    "rn-secure-storage": "^3.0.1",
```

Comment:
Are we using this package? 
If not can we remove it from package.json & *lock files?

Reply:
Done